include/powsybl/math/GraphUtil.hxx

```diff
@@ -0,0 +1,36 @@
+/**
+ * Copyright (c) 2020, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+#ifndef POWSYBL_MATH_GRAPHUTIL_HXX
+#define POWSYBL_MATH_GRAPHUTIL_HXX
+
+#include <powsybl/math/GraphUtil.hpp>
+
+namespace powsybl {
+
+namespace math {
+
+template <typename V, typename E>
+void GraphUtil::removeIsolatedVertices(UndirectedGraph<V, E>& graph) {
```

Comment:
To be removed

Reply:
This method is used in Java, so we probably miss something. If it's not called, we probably miss a change in the Node/Breaker VL implementation.